lib/osdep/linux.c

```diff
@@ -2259,7 +2257,7 @@ static int do_linux_open(struct wif * wi, char * iface)
 	dev->arptype_in = dev->arptype_out;
 
 	if (iface_malloced) free(iface);
-	if (iwpriv) free(iwpriv);
```

Comment:
this line should be deleted/comented as iwpriv points to a static char array

Reply:
This line should be deleted/comented, agreed.
The iwpriv will not exist anymore after this patch merges. All stuff return by wiToolsPath should be free, wiToolsPath  uses malloc inside.